From 5f33c48cc5b55fcf9436f231cfb8ff44e0d096de Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Sat, 19 Feb 2011 18:43:25 +0000
Subject: [PATCH 02/72] cache.h: include gettext.h
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change cache.h to include gettext.h. In previous incarnations of the
gettext series I included gettext.h in builtin.h, then changed
builtin/*.c files to include that when they didn't already.

It was pointed out by Junio and others that it makes more sense to
have the inclusion here, since gettext is going to be used everywhere.

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 cache.h |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/cache.h b/cache.h
index 3abf895..a465f38 100644
--- a/cache.h
+++ b/cache.h
@@ -5,6 +5,7 @@
 #include "strbuf.h"
 #include "hash.h"
 #include "advice.h"
+#include "gettext.h"
 
 #include SHA1_HEADER
 #ifndef git_SHA_CTX
-- 
1.7.2.3

